runtime/vm/jvminit.c

```diff
@@ -6191,6 +6197,114 @@ shutDownHookWrapper(struct J9PortLibrary* portLibrary, U_32 gpType, void* gpInfo
 }
 #endif
 
+/**
+ * Invoke jdk.internal.misc.Signal.dispatch(int number) in Java 9 and
+ * onwards. Invoke sun.misc.Signal.dispatch(int number) in Java 8.
+ *
+ * @param vmThread pointer to a J9VMThread
+ * @param signal integer value of the signal
+ *
+ * @return void
+ */
+static void
+signalDispatch(J9VMThread *vmThread, I_32 signal) {
+	J9JavaVM *vm = vmThread->javaVM;
+	J9NameAndSignature nas = {0};
+	I_32 args[] = {signal};
+
+	nas.name = (J9UTF8 *)&j9_dispatch;
+	nas.signature = (J9UTF8 *)&j9_int_void;
+
+	enterVMFromJNI(vmThread);
+
+	if (J2SE_VERSION(vm) >= J2SE_19) {
+		runStaticMethod(vmThread, (U_8 *)"jdk/internal/misc/Signal", &nas, 1, (UDATA *)args);
+	} else {
+		runStaticMethod(vmThread, (U_8 *)"sun/misc/Signal", &nas, 1, (UDATA *)args);
+	}
+
+	internalExceptionDescribe(vmThread);
```

Comment:
What is the reasoning behind printing the exception to the console.

Reply:
`sidecarExit` invokes `internalExceptionDescribe` after a call to `Shutdown.exit` (Java method). based upon existing usage, i thought it is needed after a call to Java method. should we remove `internalExceptionDescribe` from the above change?

Reply:
removed `internalExceptionDescribe`.